### DISCUSSION TITLE
How to specify connection's port?

### DISCUSSION THREAD

**Discussion by scape7yu** [SAP/PyRFC#294] [Q&A] [answered]
```python
SYSNR = '00'

with Connection(ashost=ASHOST, sysnr=SYSNR, client=CLIENT, user=USER, passwd=PASSWD, lang=LANGUAGE) as CONN:
    data = CONN.call(func_name="XXX", options={})
```

The connection's port will be 33 + SYSNR, so it's 3300 here.

Is anyone can tell me how to specify connection's port? Thanks very much.

**Comment by bsrdjan** [comment, marked as answer]
Yes, the connection port is determined by system number. 

How to set RFC connection ports is documented in `sapnwrfc.ini` file, located in SAP NWRFC SDK `demo` folder. Some examples can be found also in PyRFC issues by searching for "connection port", like https://github.com/SAP/PyRFC/issues/202#issuecomment-709968797 or  https://github.com/SAP/PyRFC/issues/212#issuecomment-734447033 ...

**Reply by scape7yu** [reply]
Thank you for your patient answer!
